src/mbgl/renderer/layers/fill_layer_tweaker.cpp

```diff
@@ -52,16 +55,19 @@ void FillLayerTweaker::execute(LayerGroupBase& layerGroup,
     auto UpdateFillUniformBuffers = [&]() {
         if (fillUniformBufferUpdated) return;
 
+#if MLN_RENDER_BACKEND_METAL
```

Comment:
Not worth doing on this one, but might be nice to make a more generic hasPermutation check.

Reply:
There's more pending, but tied up with render targets.

https://github.com/maplibre/maplibre-native/blob/6f6970ebf48532f526a12b62a27e3cd202dba844/include/mbgl/renderer/layer_tweaker.hpp#L42-L46

Reply:
Gotcha.